roles/postfix-mail-relay/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: "Install postfix and mailutils"
+  apt:
+    pkg:
+      - mailutils
+      - postfix
+    state: "present"
+    update_cache: yes
```

Comment:
update_cache is done at the beginning of the playbooks in the common role. Is it necessary again here?